freqtrade/exchange/binance.py

```diff
@@ -89,3 +94,44 @@ def stoploss(self, pair: str, amount: float, stop_price: float, order_types: Dic
                 f'Could not place sell order due to {e.__class__.__name__}. Message: {e}') from e
         except ccxt.BaseError as e:
             raise OperationalException(e) from e
+
+    def _apply_leverage_to_stake_amount(self, stake_amount: float, leverage: float):
+        return stake_amount / leverage
+
+    def fill_leverage_brackets(self):
+        """
+            Assigns property _leverage_brackets to a dictionary of information about the leverage
+            allowed on each pair
+        """
+        leverage_brackets = self._api.load_leverage_brackets()
```

Comment:
Methods that call api's (this one is supposed to call the "setApi" endpoint) MUST be wrapped in the default ccxt errorhandlers (copy/paste them from one of the existing methods ...) to avoid random errors.

obviously, that'll also require tests for each of these.

Reply:
I updated it, is that kinda what you meant?

Reply:
ye